cgnsutilities/cgnsutilities.py

```diff
@@ -1858,7 +1863,7 @@ def extrude(self, direction):
         """Extrudes from 2D panar grid to 3D"""
 
         # Get the data order and new dims
-        order, newDims = self._extrudeGetDataOrderAndDIms(direction)
+        order, newDims = self._extrudeGetDataOrderAndDIms(direction, 1)
```

Comment:
This is conflicting, and step value should be 2.

Reply:
fixed